files/__main__.py

```diff
@@ -24,6 +24,9 @@
 app.jinja_env.auto_reload = True
 faulthandler.enable()
 
+from dotenv import load_dotenv
+load_dotenv("env")
```

Comment:
these are loaded automatically by docker but I needed to load them here for the test